### ISSUE TITLE
bootstrap: make sure we don't probe ourselfs for the seeds

### ISSUE THREAD

**Issue by ktoso** [akka/akka-management#56]
The probing process is started for every discovered service node, which also means "us", but that's not very useful and we should avoid pinging ourselfes with http queries to our own contact point

**Comment by ktoso** [comment]
Will be resolved in my PR already

**Comment by ktoso** [comment]
My  chat notes

> eh the test gets weird since all nodes are 127.0.0.1
> and since contact points don’t return any seed nodes
> so we don’t “know” the ports for remoting for the other nodes
> what has to happen is the coordinator gather from the contact points all their selfAddresses and sort based on those
> and not the ports it found via the contact points thing I guess?
> when IPs are different it works

**Comment by patriknw** [comment]
I don't think it hurts much to probe ourself and it would add a some complexity to make self special.
Anyhow the same info should be included to the JoinDecider that was introduced in https://github.com/akka/akka-management/pull/154

**Comment by ktoso** [comment]
Yeah, we can ignore this I think.

**Comment by chbatey** [comment]
Re-opening this as in some environments, such as when Itsio is enabled, don't allow nodes to communicate with themselves.

There's already logic for seeing if the self node is in the list of contact points returned from discovery: https://github.com/akka/akka-management/blob/master/cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/SelfAwareJoinDecider.scala#L57

/cc @TimMoore @jroper @patriknw  does this make sense? If so it would be trivial to disable self probing, perhaps behind a flag for small clusters to reach min contact points

**Comment by TimMoore** [comment]
Was this resolved? @jroper how did you handle this in your Istio testing?
